tt_eager/tt_dnn/op_library/bmm/kernels/dataflow/reader_bmm_tile_layout_in0_sender_dram_sharded.cpp

```diff
@@ -138,5 +143,27 @@ void kernel_main() {
             cb_push_back(cb_id_in0, in0_block_num_tiles);
 
         }
+    } else { // mcast receiver + compute
```

Comment:
Does this code supposed to cover the remaining two worker_core_type values, 0 and 3?

Reply:
only for case 3, for 0 it has been checked at the beginning
```
// RUNTIME ARGS
    const uint32_t worker_core_type                               = get_arg_val<uint32_t>(0);
    // if not worker core, skip
    if (worker_core_type == 0) {
        return;
    }
```

Reply:
Thanks